SnapMD.ConnectedCare.ApiModels/ScheduledConsultationResult.cs

```diff
@@ -1,14 +1,36 @@
-﻿namespace SnapMD.ConnectedCare.ApiModels
-{
+﻿//    Copyright 2015 SnapMD, Inc.
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//        http://www.apache.org/licenses/LICENSE-2.0
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+using System;
 
-    /// <summary>
-    /// Wraps a consultation ID in a response for scheduled consultation POST APIs.
-    /// </summary>
+namespace SnapMD.ConnectedCare.ApiModels
+{
     public class ScheduledConsultationResult
     {
-        /// <summary>
-        /// The ID of the newly-created consultation.
-        /// </summary>
+        public DateTimeOffset ScheduledTime { get; set; }
+        public int ExpiryTime { get; set; }
```

Comment:
I wanted to ask you if we really use this model and how it got there. I haven't found any usages.